### DISCUSSION TITLE
The 100Devs Community Marketplace is LIVE!!

### DISCUSSION THREAD

**Discussion by Michael-Futral** [CurrentlyUnderConstruction/.github#6] [Announcements]
Let us know what you would like to see, how we can make the community better, or any issues you are having with the organization.

-100Devs Marketplace Team

**Comment by d-herz** [comment]
Woot! I was thinking it would be great if we had some sort of weekly schedule that shows the different days/times for 100Devs meetups and events. Like the various Banki meetups, or discussions that are open for anyone to partake in. The standups, stream-team streams, office hours times, etc etc.  Just so the information could be all available in one place? ~~Not sure yet what the best method to implement it would be though~~ Could do it with markdown? (See below)

**Reply by d-herz** [reply]
Something like:

| Sunday 11/20/22                 	| Monday 11/21/22                                                                                                                                                              	| Tuesday 11/22/22                                                                                                                                                     	| Wednesday 11/23/22                                                                                                                                                                                                                                    	| Thursday 11/24/22                                                                                                                                                    	| Friday 11/25/22                                                                                                                          	| Saturday 11/26/22 	|
|---------------------------------	|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|----------------------------------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|----------------------------------------------------------------------------------------------------------------------------------------------------------------------	|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------	|
| Office Hours: 2pm EST (Discord) 	| Rufio Stream: 11:00am EST (Twitch)<br><br>Banki Voice Group 1: 3:00pm EST (Discord)<br><br>Banki Voice Group 2: 5:00pm EST (Discord)<br><br>Daily Standup: 6:00pm EST (Remo) 	| Banki Voice Group 1: 3:00pm EST (Discord)<br><br>Banki Voice Group 2: 5:00pm EST (Discord)<br><br>Daily Standup: 6:00pm EST (Remo)<br><br>Class: 6:30pm EST (Twitch) 	| Banki Voice Group 1: 3:00pm EST (Discord)<br><br>Banki Voice Group 2: 5:00pm EST (Discord)<br><br>Daily Standup: 6:00pm EST (Remo)<br><br>Mock Interview Practice: 6:30pm EST (After standup) (Discord)<br><br>Mayanwolfe Stream: 7:00pm EST (Twitch) 	| Banki Voice Group 1: 3:00pm EST (Discord)<br><br>Banki Voice Group 2: 5:00pm EST (Discord)<br><br>Daily Standup: 6:00pm EST (Remo)<br><br>Class: 6:30pm EST (Twitch) 	| Banki Voice Group 1: 3:00pm EST (Discord)<br><br>Banki Voice Group 2: 5:00pm EST (Discord)<br><br>Friday Tea Spill: 6:00pm EST (Discord) 	|                   	|

**Reply by Michael-Futral** [reply]
Definitely think this would be a great addition to the organization!! Try and make a pull request whenever you want to, and let me know if you run into any issues. Will see if I can make it so you can approve pull requests for this repo yourself so you can keep that feature updated easy if you want to maintain it.

Really think if we get many people involved and in control of features in the organization; really think this would be a great thing for the 100Devs community that would be built and controlled by members as it expands.

**Reply by d-herz** [reply]
Awesome, sounds good! I'll keep an eye on events and update as we go. It'll be nice to have a clear picture of each weeks happenings. 

And I agree! I'm excited to help out. I really like the concept and what you've started here. I think it could provide a lot of value for the community and will only get better with time 😎.

**Reply by JoeDravarol** [reply]
Thanks for making this! Looking forward to seeing it grow.

@d-herz There's a Hanging out but quietly event/standup every Sunday at 12:30 EST.

**Reply by d-herz** [reply]
Good to know @JoeDravarol ! Appreciate the heads up, I will add it to the weekly schedule for sure. I might even want to attend that as well now that I am aware of it!

**Comment by experiencethebalanceoftheyinandyang** [comment]
hi

**Comment by ghost** [comment]
Woohoo I'm so excited for this 🤩

**Comment by ghost** [comment]
I think it would be awesome if we made like repos focused on learning certain subjects like basic JS, HTML Sementics, Async Await, Tailwind+ EJS templates. 

Like educational templates so ppl can see how each subject works and try it out on their VSCode. For like future beginner dev. Just a thought :) ❤️

**Reply by Michael-Futral** [reply]
That is a great idea!!! The collaboration of this community to educate fellow members and mentally grow is the entire goal! There have been class-promoted free educational resources that focus on specific programming topics, and if someone consolidated these recourses to promote them to the community it would be an amazing addition. Not sure if our team can focus on that specifically now, but if you would like to work on a repo that spotlights these free class-promoted course links; we would be happy to host it here for everyone to share.

**Reply by ghost** [reply]
Yea of course! I'm trying to think of other ideas that could help haha ❤️
